auth/__init__.py

```diff
@@ -0,0 +1,102 @@
+from typing import Dict, Set
+
+import connexion
+from decorator import decorator
+from jose import jwt
+
+from auth.AuthConstants import EC_LEADERSHIP_ROLE, NATIONAL_REPORT_GENERATOR_ROLE, NATIONAL_REPORT_VIEWER_ROLE
+from orm.entities.Submission import TallySheet
+from orm.entities.Submission.TallySheet import TallySheetModel
+
+JWT_SECRET = "jwt_secret"
+CLAIM_PREFIX = "areaAssignment/"
+AREA_ID = "areaId"
+
+
+def get_unauthorized_msg(detail):
+    return connexion.problem(401, "Unauthorized", detail)
```

Comment:
There's an exception class I have defined. May be we could use that

Reply:
Fix with 066207d